specification/validated-relationships-service-api.yaml

```diff
@@ -317,57 +322,221 @@ components:
+                required: 
+                  - relatedPerson_name
+                properties:
+                  relatedPerson_name:
+                    type: array
+                    items:
+                      oneOf:
+                        - type: object
+                          required:
+                            - linkId
+                            - text
+                            - answer
+                          properties:
+                            linkId:
+                              type: string
+                              enum: ["relatedPerson_name_given"]
+                            text:
+                              type: string
+                              example: "Given names, including any middle names."
+                            answer:
+                              type: array
+                              items:
+                                oneOf:
+                                  - type: object
+                                    properties:
+                                      valueString:
+                                        type: array
+                                        description: |
+                                          Each name(s) should be a separate item in the list. The first given name may include multiple names, separated by a space.
+                                          Subsequent names must be broken down into list items. For example, the input `[Jane Marie Anne, Jo Adele]` returns `[Jane Marie Anne, Jo, Adele]`.
+
+                                        maxItems: 5                                  
+                                        example: [Sharon Melissa]
+                                        items:
+                                          type: string
+                                          maxLength: 35
+                                          example: Sharon
+
+                        - type: object
+                          required:
+                            - linkId
+                            - text
+                            - answer
+                          properties:
+                            linkId:
+                              type: string
+                              enum: ["relatedPerson_name_family"]
+                            text:
+                              type: string
+                              example: "Family name (often called Surname)."
+                            answer:
+                              type: array
+                              minItems: 1
+                              maxItems: 1
+                              items:
+                                oneOf:
+                                  - type: object
+                                    properties:
+                                      valueString:
+                                        type: string
+                                        example: "Smith"
+
+              - type: object
```

Comment:
`relatedPerson_birthDate` will only be mandatory if a patient is nominating. Likely will have to check this through logic.

Reply:
Removed the required links for name and DOB due to above logic depending on who is the applicant